caddyhttp/proxy/proxy_test.go

```diff
@@ -247,8 +247,8 @@ func TestWebSocketReverseProxyNonHijackerPanic(t *testing.T) {
 func TestWebSocketReverseProxyServeHTTPHandler(t *testing.T) {
 	// No-op websocket backend simply allows the WS connection to be
 	// accepted then it will be immediately closed. Perfect for testing.
-	var connCount int32
```

Comment:
@mholt I think we could still make sure only one connection is made here: If there are two connections reaching the backend, the second one will trigger a panic due to close of a closed channel, the test will fail as a result. Also, if there is no connection issued, the test will also fail because of timeout.